test/unit/constraints/array.test.ts

```diff
@@ -0,0 +1,113 @@
+            class MyArrayClass {
+                @Max(5)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [3,2,3,3];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).toBeNull();
+        })
+        it('Should allow correct max value for array - Negative Test', () => {
+            class MyArrayClass {
+                @Max(1)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [3,2,3,3];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).not.toBeNull();
+        })
+        it('Should allow the correct array length', () => {
+            class MyArrayClass {
+                @Length(4)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [3,2,3,3];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).toBeNull();
+        })
+        it('Should allow the correct array length - Negative Test', () => {
+            class MyArrayClass {
+                @Length(3)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [3,2,3,3];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).not.toBeNull();
+        })
+        it('Should allow the correct minimum value for Array', () => {
+            class MyArrayClass {
+                @Min(4)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [6,7,8,9];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).toBeNull();
+        })
+        it('Should allow the correct minimum value for Array - Negative Test', () => {
+            class MyArrayClass {
+                @Min(5)
+                @ArraySchema()
+                myArray!: Array<number>
+            }
+            let object = new MyArrayClass();
+            object.myArray = [2,3,4];
+            const validator = new Validator();
+            const result = validator.validateAsClass(object, MyArrayClass);
+            expect(result.error).not.toBeNull();
+        })
+    })
+    describe("Unique", function () {
```

Comment:
👍 Nice work! This test scenario is concise and follows the codebase conventions.